.github/workflows/ci.yml

```diff
@@ -71,7 +71,7 @@ jobs:
                 uses: shivammathur/setup-php@v2
                 with:
                     coverage: "none"
-                    php-version: "8.2"
+                    php-version: "8.3"
```

Comment:
please keep those to `8.2` for now. As the only contributor to the rewrite happening in the `main` branch currently, it is a lot easier if the phpstan job in the CI uses the same PHP version that the one I use locally (sometimes, the output of phpstan depends on which PHP version was used to run it).

Reply:
Done.